tests/test_result.py

```diff
@@ -10,14 +10,14 @@
 from tox.result import ResultLog
 
 
-@pytest.fixture
-def pkg(tmpdir):
-    p = tmpdir.join("hello-1.0.tar.gz")
-    p.write("whatever")
-    return p
+@pytest.fixture(name="pkg")
```

Comment:
should we rename the fixture to fake_pkg while at it?

Reply:
Good idea, but also better done later in a another refactoring step.